.taskcluster.yml

```diff
@@ -6,301 +6,285 @@ policy:
+        project:
+            $if: 'tasks_for == "github-push"'
+            then: '${event.repository.name}'
+            else:
+                $if: 'tasks_for == "github-pull-request"'
+                then: '${event.pull_request.head.repo.name}'
+                else:
+                    $if: 'tasks_for in ["cron", "action"]'
+                    then: '${repository.project}'
+        head_branch:
+            $if: 'tasks_for == "github-pull-request"'
+            then: ${event.pull_request.head.ref}
+            else:
+                $if: 'tasks_for == "github-push"'
+                then: ${event.ref}
+                else:
+                    $if: 'tasks_for in ["action", "cron"]'
+                    then: '${push.branch}'
+        head_sha:
+            $if: 'tasks_for == "github-push"'
+            then: '${event.after}'
+            else:
+                $if: 'tasks_for == "github-pull-request"'
+                then: '${event.pull_request.head.sha}'
+                else:
+                    $if: 'tasks_for in ["action", "cron"]'
+                    then: '${push.revision}'
+        ownTaskId:
+            $if: '"github" in tasks_for'
+            then: {$eval: as_slugid("decision_task")}
+            else:
+                $if: 'tasks_for in ["cron", "action"]'
+                then: '${ownTaskId}'
+        pullRequestAction:
+            $if: 'tasks_for == "github-pull-request"'
+            then: ${event.action}
+            else: 'UNDEFINED'
       in:
-          $let:
-              # Github events have this stuff in different places...
-              ownerEmail:
-                  $if: 'tasks_for in ["cron", "action"]'
-                  then: '${tasks_for}@noreply.mozilla.org'
-                  else:
-                      $if: 'event.sender.login == "bors[bot]"'
-                      then: 'skaspari+mozlando@mozilla.com'   # It must match what's in bors.toml
-                      else:
-                          $if: 'tasks_for == "github-push"'
-                          then: '${event.pusher.email}'
+        $if: >
+            tasks_for in ["action", "cron"]
+            || (tasks_for == "github-pull-request" && pullRequestAction in ["opened", "reopened", "synchronize"])
+            || (tasks_for == "github-push" && head_branch[:10] != "refs/tags/") && (head_branch != "staging.tmp") && (head_branch != "trying.tmp") && (head_branch[:8] != "mergify/")
+        then:
+            $let:
+                level:
+                    $if: 'tasks_for in ["github-push", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/fenix"'
+                    then: '3'
+                    else: '1'
+                short_head_branch:
+                    $if: 'head_branch[:11] == "refs/heads/"'
+                    then: {$eval: 'head_branch[11:]'}
+            in:
+                taskId:
+                    $if: 'tasks_for != "action"'
+                    then: '${ownTaskId}'
```

Comment:
I think this will be slightly different as the `taskId` key will exist and be empty for action tasks? Not sure if that's a problem or not.

Reply:
I tested it locally, the `taskId` doesn't seem to be there at all when declared like that.
In order to make it "empty" I think you'd have to set `else: ""`